Source/ui/UI.js

```diff
@@ -33,8 +33,13 @@ export function message(text, document) {
  * by pressing the OK button.
  */
 export function alert(title, text) {
-  const app = NSApplication.sharedApplication()
-  app.displayDialog_withTitle(text, title)
+  const dialog = NSAlert.alloc().init()
+  dialog.setMessageText(title)
+  dialog.setInformativeText(text)
+  if (__command.pluginBundle() && __command.pluginBundle().icon()) {
+    dialog.icon = __command.pluginBundle().icon()
```

Comment:
If the plugin hasn't supplied it's own icon would this still default to the sketch icon? Wondering if we need an `else` to set the plugin icon here instead?

Reply:
Ah yes that'd be great indeed, good call